src/UI/Tester/WB.UI.Tester/Activities/PrefsActivity.cs

```diff
@@ -51,6 +51,8 @@ public override void OnCreatePreferences(Bundle savedInstanceState, string rootK
                 this.SetPreferenceTitleAndSummary(TesterSettings.DesignerEndpointParameterName, TesterUIResources.Prefs_DesignerEndPointTitle, settings.Endpoint);
                 this.SetPreferenceTitleAndSummary("AcceptUnsignedSslCertificate", TesterUIResources.Prefs_AcceptUnsignedTitle, TesterUIResources.Prefs_AcceptUnsignedSummary);
 
+                this.SetPreferenceTitleAndSummary(TesterSettings.TestApplicationCrash, "Crash application", string.Empty);

```

Comment:
Is it need in real app? remove it if not

Reply:
It's a trigger that helps to test this reporting functionality.